tooling/generate-registry-files.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const didMethodRegex = /<tr>\n.*<td>\n\s+did:(.*):\n.*<\/td>\n.*<td>\s+(.*)\n.*<\/td>\n.*<td>\s+(.*)\n.*<\/td>\n.*<td>\n\s+(.*)\n.*<\/td>\n.*<td>\n\s+(.*)\n.*/g;
+const didSpecRegex = /<a href="(.*)">.*<\/a>/;
+const websiteRegex = /<a href="(.*)">.*<\/a>/;
+const emailRegex = /<a href="mailto:(.*)">.*<\/a>/;
+// set directories and paths
+const registryFile = path.join(__dirname, '../index.html');
+const methodsDir = path.join(__dirname, '../methods');
+const sectionDir = path.join(__dirname, '../sections');
+
+// create directories
+mkdirp.sync(methodsDir);
+mkdirp.sync(sectionDir);
+
+const validateRegistryEntry = require('./validation');
+
+// we migth consider not allowing html here
+const getUnsafeContact = (contact)=>{
+  // const obj = {}
+  // if (contact.includes('href="http')){
+  //   let website = contact.match(websiteRegex)[1];
+  //   obj.website = website;
+  // }
+  // if (contact.includes('href=\"mailto')){
+  //   let email = contact.match(emailRegex)[1];
+  //   obj.email = email;
+  // } 
+  // // no useful structure
+  // if (!Object.keys(obj).length){
+  //   obj.name = contact
+  // }
+  // // flatten object to single string value.
+  // if (obj.website){
+  //   return obj.website
+  // }
+  // if (obj.email){
+  //   return obj.email
+  // }
+  // return obj.name 
+  return contact;
+}
+
+// we migth consider not allowing html here
+const getUnsafeRegistry = (registry)=>{
```

Comment:
We might also consider removing the registry field, many entries are not correct, or have provided basically unhelpful values here... now is the time to remove this field if we want to.... I thing this issue is better handled elsewhere.

Reply:
registry is the column previously known as DLT or Network?